### DISCUSSION TITLE
My Strongswan site to site vpn connection keeps giving peer not responding when trying to establish connection

### DISCUSSION THREAD

**Discussion by debashish-github** [strongswan/strongswan#404] [Q&A]
Hi,   I  have set up two sites for a site to site vpn connection. I have setup configuration as attached and added them to the ipsec.conf and ipsec.secrets files .
The when I try to run strongswan up <connectionName> it fails to establish connection with the following dump:
initiating Main Mode IKE_SA NuanceInterop-mirth[1] to 52.136.121.150
generating ID_PROT request 0 [ SA V V V V V ]
sending packet: from 52.191.194.199[500] to 52.136.121.150[500] (252 bytes)
sending retransmit 1 of request message ID 0, seq 1
sending packet: from 52.191.194.199[500] to 52.136.121.150[500] (252 bytes)
sending retransmit 2 of request message ID 0, seq 1
sending packet: from 52.191.194.199[500] to 52.136.121.150[500] (252 bytes)
sending retransmit 3 of request message ID 0, seq 1
sending packet: from 52.191.194.199[500] to 52.136.121.150[500] (252 bytes)
sending retransmit 4 of request message ID 0, seq 1
sending packet: from 52.191.194.199[500] to 52.136.121.150[500] (252 bytes)
sending retransmit 5 of request message ID 0, seq 1
sending packet: from 52.191.194.199[500] to 52.136.121.150[500] (252 bytes)
giving up after 5 retransmits
peer not responding, trying again (2/3)

[NuanceInterop.zip](https://github.com/strongswan/strongswan/files/6497029/NuanceInterop.zip)
I am really stuck up with this . Can someone help me to figure out the root cause ?
Thanks

**Comment by tobiasbrunner** [comment]
This is the initial IKE message, so could be all sorts of basic things, like the wrong IPs, a firewall blocking the traffic, some generic network/routing issue. So please double check all of that (also check the remote log). Also, please don't use IKEv1 between two instances of strongSwan (and using weak algorithms is also not necessary).

**Comment by debashish-github** [comment]
Thanks for your sugestion. There was indeed a misconfiguration. FIXED now !
